packages/flutter/lib/src/material/ink_splash.dart

```diff
@@ -75,6 +75,50 @@ class _InkSplashFactory extends InteractiveInkFeatureFactory {
   }
 }
 
+abstract class InkSplashRippleMixin {
```

Comment:
The comment in line 79, 80 will not be needed if we change `abstract class` to `mixin`